tests/conftest.py

```diff
@@ -138,7 +138,7 @@ def _verify_db_contents():
             return (True, None)
 
         # Verify that ASIC DB has been fully initialized
-        init_polling_config = PollingConfig(2, 30, strict=True)
+        init_polling_config = PollingConfig(2, 60, strict=True)
```

Comment:
Why we change this value from 30 to 60?

Reply:
The timeout duration was increased after analyzing the failures and identifying that the issue might be related to initialization timing. Adding a delay helped resolve the problem, and the build was successful. However, when these delays were removed, the dvs_test failed again, indicating that the delays are necessary for the DVS to be ready.

Reply:
Please narrow down the failure and use fine-tuned sleep intervals. These can increase vstest execution times.

Reply:
fine tuned